### ISSUE TITLE
SA1121 (UseBuiltInTypeAlias) code fix does not work inside documentation comments

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#620]
While SA1121 is reported for the following case, the code fix does not provide an automatic correction:

```csharp
/// <see cref="System.String"/>
```
